content/_layouts/developerguide.html.haml

```diff
@@ -1,5 +1,9 @@
 ---
-layout: developer
+layout: developerbook
 ---
 
+%p
+  %a{:href => '../../guides'}
+    View the index of available developer guides
```

Comment:
The placement of this is a little wacky (under the page title), but I don't have a better suggestion 🤷‍